app/views/events/_table_badges.html.erb

```diff
@@ -0,0 +1,30 @@
+<span class="badge"><%= table.scenario.tier %></span>
```

Comment:
That's just how Rails templating works. There are ways, but it would be ... less appealing.